### PR TITLE
WEST MIDLANDS CLASS 3 - GEORGINA ASHUN  - GIT/GITHUB - WEEK 1

### PR DESCRIPTION
**Pull request by gynaa78**
# Your Details

- Your Name:Georgina Ashun
- Your City: Telford
- Your Slack Name: Georgina Ashun

# Homework Details

- Module: Git/Github
- Week: 1

-----
[View rendered .github/pull_request_template.md](https://github.com/gynaa78/GitHomeworkFixErrors/blob/main/.github/pull_request_template.md)
[View rendered homework.md](https://github.com/gynaa78/GitHomeworkFixErrors/blob/main/homework.md)

### CODE REVIEW

File: homework.md [hunk 1]
```diff
@@ -44,70 +44,72 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
 
-A place for us to store and share our code.
+A web service to store and share our code using Git.
 
 ## Question 15
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+To save changes to a local repo
```
Comment (tenzyns) [inline homework.md:97]: it can save changes either to a local repo or remote gitHub web.

File: homework.md [hunk 2]
```diff
@@ -44,70 +44,72 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
 
 ## Question 16
 
 What program do we use to write code at CodeYourFuture?
 
-Microsoft Word
+HTML, CSS, JavaScript
```
Comment (tenzyns) [inline homework.md:103]: CodeYourFuture use Visual Studio Code for writing codes.

File: homework.md [hunk 3]
```diff
@@ -44,70 +44,72 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
 
 ## Question 17
 
 What do you use "Commit Messages" for?
 
-To record what changes you made to the code
+To record and save what changes you made to the code in the local repo after staging in Git
```
Comment (tenzyns) [inline homework.md:109]: It is to let others know what commits you have made and why.

**Review by tenzyns — COMMENTED** [review]
I have completed the review.

**Comment by Halla-S** [comment]
well done ,Georgina

**Comment by Halla-S** [comment]
well done ,Georgina

**Review by MireilleNyoubissi — COMMENTED** [review]
Good overall. After Tenzys correction, there is nothing left to say:)

**Pull request closed without merging**